assets/images/logo/speed-dial/README.md

```diff
@@ -0,0 +1,13 @@
+# Speed Dials
```

Comment:
Sorry, I still just don’t see the point. Is it only scaled-up logo with background, or is there something more?

Reply:
Yes, just logo with background. Do you want them to be taken out?

Reply:
Maybe. But the question was caused mostly by me worrying about the repo size, so if you fix the sharpness &/ size, they can prolly stay in.